app/javascript/admin/dashboard/index.js

```diff
@@ -1,5 +1,68 @@
 import Chart from 'chart.js';
 
+function createVisitsChart(id, title, dataSource) {
```

Comment:
When you create a chart, would you continue to use `createChart` to keep things DRY? Please edit the `backgroundColor` and `borderColor` given you have more colors.

Reply:
Yes, Fixed